tests/acceptance/features/bootstrap/ActivityContext.php

```diff
@@ -70,12 +72,53 @@ public function activityWithIndexShouldMatch(
 		}
 	}
 
+	/**
+	 * @Then the activity number :index of user :user should match these properties:
+	 *
+	 * @param string $index (starting from 1, newest to the oldest)
+	 * @param string $user
+	 * @param TableNode $expectedProperties
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function activityWithIndexShouldMatch(
+		string $index,
+		string $user,
+		TableNode $expectedProperties
+	): void {
+		$responseDecoded = $this->getActivitiesForUser($user);
+		$this->assertActivityIndexContent($user, $index, $responseDecoded, $expectedProperties);
+	}
+
+	/**
+	 * @Given /^as user "([^"]*)" the activity number (\d+) for "([^"]*)" should match these properties:$/
```

Comment:
```suggestion
	 * @Then /^as user "([^"]*)" the activity number (\d+) for "([^"]*)" should match these properties:$/
```

This looks like a Then step.

Reply:
fixed